workflow-service/src/test/java/com/redhat/parodos/workflow/registry/BeanWorkFlowRegistryImplTest.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright (c) 2022 Red Hat Developer
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.parodos.workflow.registry;
+
+import com.redhat.parodos.workflow.definition.entity.WorkFlowTaskDefinitionEntity;
+import com.redhat.parodos.workflows.definition.WorkFlowCheckerDefinition;
+import org.mockito.Mockito;
+
+import java.util.*;
+
+import org.junit.Test;
+import org.junit.Before;
+import com.redhat.parodos.workflows.common.enums.WorkFlowType;
+import com.redhat.parodos.workflows.definition.WorkFlowDefinition;
+import com.redhat.parodos.workflows.definition.task.WorkFlowTaskDefinition;
+import com.redhat.parodos.workflows.workflow.WorkFlow;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.redhat.parodos.workflow.definition.repository.WorkFlowDefinitionRepository;
+import com.redhat.parodos.workflow.definition.repository.WorkFlowTaskDefinitionRepository;
+import com.redhat.parodos.workflow.definition.entity.WorkFlowDefinitionEntity;
+
+import static org.junit.Assert.*;
+
+public class BeanWorkFlowRegistryImplTest {
+    private  WorkFlowDefinitionRepository wfDefinitionRepo;
+    private WorkFlowTaskDefinitionRepository wfTaskDefinitionRepo;
+    private List<WorkFlowDefinition> wfDefinitions;
+    private List<WorkFlowTaskDefinition> wfTaskDefinitions;
+    private List<WorkFlow> wfExecutions;
+
+    // helper functions
```

Comment:
Why not to make the functions private? Do we want to reuse them?

Reply:
You are correct